build.sh

```diff
@@ -7,6 +7,11 @@ set -x
 # only exit with zero if all commands of the pipeline exit successfully
 set -o pipefail
 
+if [ ! -f "$1" ]; then
+    echo "$1 is not a file"
+    exit 1
+fi
+
```

Comment:
Why is this necessary?

Reply:
This is not necessary at all for this feature. 
But it would be great to avoid stuffing the directory with many empty files if it does not receive a correct file as the first argument.

Reply:
This change defeats the purpose of defaulting to `example.jsonnet`. I am curious how did you end up with empty files cause it shouldn't happen. That said even if we wanted to add this improvement, I would prefer if we did so in a separate PR since it is unrelated.